jmespath-core/src/main/java/io/burt/jmespath/BaseRuntime.java

```diff
@@ -29,25 +32,23 @@
     '\"', '\"'
   );
 
+  private final RuntimeConfiguration configuration;
   private final FunctionRegistry functionRegistry;
   private final NodeFactory<T> nodeFactory;
 
   /**
    * Create a new runtime with a default function registry.
    */
   public BaseRuntime() {
-    this(null);
+    this(RuntimeConfiguration.defaultConfiguration());
   }
 
   /**
-   * Create a new runtime with a custom function registry.
+   * Create a new runtime with configuration.
```

Comment:
It's not a reference to any configuration class, it's more of a general statement "Create a runtime with a thing that is configuration", not "Create a runtime with an instance of RuntimeConfiguration".